erikbenedict/minorly-unique-text-editor
# feature: updated README.md

## TEST

- proofread README.md for spelling errors
- click on all links to ensure functionality
- see that screenshot displays correctly
